libraries/AP_HAL_Linux/HAL_Linux_Class.cpp

```diff
@@ -437,6 +444,16 @@ void HAL_Linux::setup_signal_handlers() const
     sigaction(SIGINT, &sa, NULL);
 }
 
+void HAL_Linux::set_cpu_affinity(const char *str) const 
```

Comment:
there is no need to route this through HAL just o get back into linux-only. I explicitly avoided that in the version I pushed

Reply:
done